### ISSUE TITLE
Plugin no longer work

### ISSUE THREAD

**Issue by NoAccount1** [BrainMaestro/cerebro-stackoverflow#10]
Using Cerebro 0.3.0

The plugin seems to not working because of a API change 

![image](https://user-images.githubusercontent.com/74969657/137583342-2c64dfd6-d24c-4340-8a4a-ec7968895e9b.png)

**Comment by BrainMaestro** [comment]
Yeah that was bound to happen eventually. I am not supporting this plugin anymore though. Feel free to fork and continue working on it if you wish
